README.md

```diff
@@ -207,6 +207,24 @@ Type reset to start again and test this by adding the phrase "to eat there...":
 
 # Troubleshooting
 
+* Deploy using Cloud Foundry 'cf push' gives:
+
+``FAILED
+Could not find service wcsi-conversation-service to bind to watson-conversation-slots-intro``
+
+You can still use `cf push`, but realize that it is trying to bind to the services listed in the manifest.yml
+
+So, there are 3 ways you can get this to work.
+
+    * remove the services listed in the manifest.
+    * change the names of your bluemix services to match the names in the manifest.
+    * change the names in the manifest to match the names of your bluemix services.
+
+The prefered  option is number 2. Changing the manifest means you have to change it again if you have to pull from master for any reason (bug fix, new feature, ...).
```

Comment:
Looks like there is an extra space between prefered and option.
Instead of "number 2" maybe say "the second one" since it isn't a numbered list? Meh. Not sure that is better.
s/bluemix/Bluemix/ (2 places)